src/Components/test/E2ETest/Tests/GlobalizationTest.cs

```diff
@@ -3,36 +3,31 @@
 
 using System;
 using System.Globalization;
-using BasicTestApp;
 using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
 using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
 using Microsoft.AspNetCore.E2ETesting;
 using OpenQA.Selenium;
-using OpenQA.Selenium.Support.UI;
-using TestServer;
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests
+namespace Microsoft.AspNetCore.Components.E2ETests.Tests
 {
-    // For now this is limited to server-side execution because we don't have the ability to set the
-    // culture in client-side Blazor.
-    public class GlobalizationTest : ServerTestBase<BasicTestAppServerSiteFixture<InternationalizationStartup>>
+    public abstract class GlobalizationTest<TServerFixture> : ServerTestBase<TServerFixture>
+        where TServerFixture : ServerFixture
     {
-        public GlobalizationTest(
-            BrowserFixture browserFixture,
-            BasicTestAppServerSiteFixture<InternationalizationStartup> serverFixture,
-            ITestOutputHelper output)
+        public GlobalizationTest(BrowserFixture browserFixture, TServerFixture serverFixture, ITestOutputHelper output)
             : base(browserFixture, serverFixture, output)
         {
         }
 
-        protected override void InitializeAsyncCore()
-        {
-            Navigate(ServerPathBase);
-            Browser.MountTestComponent<CulturePicker>();
-            Browser.Exists(By.Id("culture-selector"));
-        }
+        protected abstract void SetCulture(string culture);
+
+        /// <summary>
+        /// Blazor Server and these tests will use the application's UtcOffset when calculating DateTimeOffset when
+        /// an offset is not explicitly specified. Blazor WASM always calculates DateTimeOffsets as Utc.
+        /// We'll use <see cref="UtcOffset"/> to express this difference in calculating expected values in these tests.
+        /// </summary>
+        protected abstract TimeSpan UtcOffset { get; }
```

Comment:
If you do `new DateTimeOffset(new Date(2008, 01, 01))` - .NET Core uses the machine's timezone offset as the offset, Wasm always uses Utc (+0.0). Outside of this, there wasn't anything surprising with globalization.

Reply:
There's a Mono issue tracking this and we're working with them to resolve it.